plugin/evm/vm.go

```diff
@@ -273,6 +273,7 @@ func (vm *VM) Initialize(
 	ethConfig.RPCEVMTimeout = vm.config.APIMaxDuration.Duration
 	ethConfig.RPCTxFeeCap = vm.config.RPCTxFeeCap
 	ethConfig.TxPool.NoLocals = !vm.config.LocalTxsEnabled
+	ethConfig.TxPool.Locals = vm.config.PriorityRegossipAddresses
```

Comment:
More locals will be added if `NoLocals=false`

Reply:
This is just for init: https://github.com/ava-labs/coreth/blob/f59716999fd10446a91a9aa2e019203f555dbac0/core/tx_pool.go#L331-L334